pyproject.toml

```diff
@@ -42,6 +42,7 @@ requests = "2.31.0"
 watchtower = "1.0.6"
 psycopg = {version = "3.1.6", extras = ["pool"]}
 unleashclient = "^5.8.0"
+uvicorn = "^0.27.1"
```

Comment:
can be installed as connexion extras which might be a better option because it will install a compatible version with the connexion